src/components/HTMLEngineProvider/BaseHTMLEngineProvider.js

```diff
@@ -81,6 +84,7 @@ function AnchorRenderer({tnode, key, style}) {
             rel={htmlAttribs.rel || 'noopener noreferrer'}
             style={style}
             key={key}
+            text={fileName}
```

Comment:
like above, change text prop name to fileName

Reply:
Done